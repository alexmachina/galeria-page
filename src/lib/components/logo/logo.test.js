import React from "react";
import renderer from "react-test-renderer";
import Logo from "./";

describe("Components", () => {
  describe("Logo", () => {
    it("Renders", () => {
      const tree = renderer.create(<Logo />).toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
