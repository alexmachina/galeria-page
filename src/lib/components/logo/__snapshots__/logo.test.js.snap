// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Logo Renders 1`] = `
<img
  alt="Logo Galeria PagÃ©"
  className="logo-0-2-1"
  src={
    Object {
      "process": [Function],
    }
  }
/>
`;
